File: 2026-08/src/day_31/prms_120905/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

vector<int> solution(int n, vector<int> numlist) {
    vector<int> v;
    for (int x : numlist) {
        if (x % n == 0) v.push_back(x);
    }

    return v;
}

File: 2026-08/src/day_31/prms_120905/Solution.java
package day_31.prms_120905;

import java.util.*;

class Solution {
    public int[] solution(int n, int[] numlist) {
        List<Integer> list = new ArrayList<>();
        for (int x : numlist) {
            if (x % n == 0) list.add(x);
        }

        return list.stream().mapToInt(Integer::intValue).toArray();
    }
}
